build/buildCLR.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -exu
+
+RUNTIME_VERSION="6.0.0-preview.5.21301.5"
+
+
+git clone --single-branch --depth 1 -b v$RUNTIME_VERSION "https://github.com/dotnet/runtime.git"
+cd runtime
+
+./build.sh clr --clang12 --checked
```

Comment:
We'll need to output the results somewhere. If you look at the other build scripts (e.g. buildcc65) they're responsible for archiving their output and then copying that tarball somewhere (often S3), What artifacts do we need to save from this build?

Reply:
The primary build output is created in `./artifacts/bin/coreclr/Linux.x64.Checked`, see https://github.com/compiler-explorer/compiler-explorer/blob/b78dfa6e24685e327ec3d762e04e671bf60d0d46/etc/config/csharp.defaults.properties#L13.

Reply:
Also `./.dotnet` seems to be required, see https://github.com/compiler-explorer/compiler-explorer/blob/b78dfa6e24685e327ec3d762e04e671bf60d0d46/etc/config/csharp.defaults.properties#L10.

Reply:
Both `./artifacts/bin/coreclr/Linux.x64.Checked` and `./.dotnet` are created by `build.sh`, so I guess you can just tar gzip both folders and upload them to S3.

Reply:
@chausner - the idea is that if you're submitting a PR to add this, we'd hope you could do this. There are many scripts to copy from to do this. All the other build.sh scripts have a few lines at the end to copy files around.

Reply:
And...apologies @chausner ! You weren't the original PR author 😊 - I'm so sorry. I didn't mean to be so snappy here, sorry. I'll definitely help it over the finish line once I understand what's going on...